app/views/partials/_navbar.html.haml

```diff
@@ -1,3 +1,6 @@
+= notice || alert
 =link_to "The Sunrise Press", root_path
-=link_to "Registration"
-=link_to "Login"
+=link_to "Register", new_user_registration_path
```

Comment:
Can you put this into the `else` part of the `if` statement so that it only appears if you are not logged in?

Reply:
Like this
```
-if user_signed_in?
 =link_to "Logout", destroy_user_session_path, method: :delete
-else
 =link_to "Registration"
 =link_to "Login"